src/main/java/com/google/sps/data/Chat.java

```diff
@@ -72,18 +74,19 @@ public static Chat getChatById(Datastore datastore, String chatId) {
 
         Query<Entity> query = Query.newEntityQueryBuilder()
             .setKind("chat")
-            .setFilter(PropertyFilter.gt("__key__", keyFactory.newKey(chatId)))
+            .setFilter(PropertyFilter.eq("__key__", keyFactory.newKey(chatId)))
             .build();
```

Comment:
I'm not sure what `__key__` is (is that documented somewhere?) but if you're trying to get a single chat by ID you can use `datastore.get` as the [docs](https://cloud.google.com/datastore/docs/concepts/entities#retrieving_an_entity) show. A query is best when you're potentially getting multiple chats.

Reply:
Yeah it is documented, it's on the updated data store guide page when you look for how to query by a key. I took that to be a valid way to query an entity but I'll change that with get which I also saw but couldn't learn enough about

Reply:
Ah! It seems like that should work as well then 🤔

Reply:
Funny thing though, I don't think its working how I expect since Im not gettting a correct query. Im gonna try .get